src/scripts/wait-for-it.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
```

Comment:
[Q] Why do you need this file? if I understand correctly and you want to wait for postgres to be online, then there is probably a cleaner solution here 
https://github.com/peter-evans/docker-compose-healthcheck#waiting-for-postgresql-to-be-healthy